src/camera.lua

```diff
@@ -8,6 +8,7 @@ local camera = {
     scale = 2, -- pixel scale (effective viewport size)
     focus_x = 0,
     focus_y = 0,
+    max_game_width = 640, -- maximum camera width in game coordinates
```

Comment:
Can you explain what this means / how this affects the camera scaling? It obviously changes the camera size similarly to how it worked previously. Should this be changed to 'zoom' the camera?

Reply:
The value here is used to determine the scale based on the resolution. For instance, if a host had a massive screen, the scale is initially 1 to 1 (1 screen pixel is one game pixel). That would make the camera see way too far into the world, so now the scale is gradually increased until the camera precedes this width in game units.

Smaller values will zoom the camera in.